node/npm_dependencies.ts

```diff
@@ -209,6 +213,7 @@ interface VendorNPMSpecifiersOptions {
   importMap: ImportMap
   logger: Logger
   referenceTypes: boolean
+  bundleProd: boolean
```

Comment:
We already have a boolean parameter (`referenceTypes`) that we toggle based on whether we're bundling for production or local development. Rather than adding a new one, can we have a single `production: boolean` or `environment: "production" | "dev"` that we use for both?

Reply:
done in https://github.com/netlify/edge-bundler/pull/519/commits/b8394e8803849819f3e56877754f8b658a9a5c7d